### ISSUE TITLE
Inconsistencies with moves updates

### ISSUE THREAD

**Issue by GoogleCodeExporter** [todtim2/universal-pokemon-randomizer#6]
```
What steps will reproduce the problem?
1. Randomize pokemon Platinum
2. Update moves to B/W
3. Still the same

What is the expected output? What do you see instead?
For example my turtwig has tackle, and with the update the power has to be 50 
and the accuracy 100, but it still is 35 power and 95 accuracy

What version of the product are you using? On what operating system?
1.2.0 Windows Xp

Please provide any additional information below.

```

Original issue reported on code.google.com by `deadpool...@gmail.com` on 23 Feb 2013 at 5:26

**Comment by GoogleCodeExporter** [comment]
```
Issue acknowledged - Platinum move updating was not editing the right file. 
Will be fixed for next release.
```

Original comment by `dabomstew` on 25 Feb 2013 at 4:54
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the response, Any idea when is gonna be the next update?
```

Original comment by `deadpool...@gmail.com` on 25 Feb 2013 at 2:40

**Comment by GoogleCodeExporter** [comment]
```
Fixed as of now in 1.2.0a
```

Original comment by `dabomstew` on 27 Feb 2013 at 2:49
- Changed state: **Fixed**
